test-framework/derby/src/main/java/io/quarkus/test/derby/DerbyDatabaseTestResource.java

```diff
@@ -15,31 +13,33 @@ public class DerbyDatabaseTestResource implements QuarkusTestResourceLifecycleMa
 
     @Override
     public Map<String, String> start() {
-        try {
-            NetworkServerControl server = new NetworkServerControl();
-            server.start(new PrintWriter(System.out));
-            for (int i = 1; i <= NUMBER_OF_PINGS; i++) {
-                try {
-                    System.out.println("[INFO] Attempt " + i + " to see if Derby Network server started");
-                    server.ping();
-                    break;
-                } catch (Exception ex) {
-                    if (i == NUMBER_OF_PINGS) {
-                        System.out.println("Derby Network server failed to start");
-                        ex.printStackTrace();
-                        throw ex;
-                    }
-                    try {
-                        Thread.sleep(SLEEP_BETWEEN_PINGS);
-                    } catch (InterruptedException ignore) {
-                    }
-                }
-            }
-            System.out.println("[INFO] Derby database started in TCP server mode");
-        } catch (Exception e) {
-            throw new RuntimeException(e);
-        }
-        return Collections.emptyMap();
+        throw new NullPointerException("dummy");
+
+        //        try {
+        //            NetworkServerControl server = new NetworkServerControl();
+        //            server.start(new PrintWriter(System.out));
+        //            for (int i = 1; i <= NUMBER_OF_PINGS; i++) {
+        //                try {
+        //                    System.out.println("[INFO] Attempt " + i + " to see if Derby Network server started");
+        //                    server.ping();
+        //                    break;
+        //                } catch (Exception ex) {
+        //                    if (i == NUMBER_OF_PINGS) {
+        //                        System.out.println("Derby Network server failed to start");
+        //                        ex.printStackTrace();
+        //                        throw ex;
+        //                    }
+        //                    try {
+        //                        Thread.sleep(SLEEP_BETWEEN_PINGS);
+        //                    } catch (InterruptedException ignore) {
+        //                    }
+        //                }
+        //            }
+        //            System.out.println("[INFO] Derby database started in TCP server mode");
+        //        } catch (Exception e) {
+        //            throw new RuntimeException(e);
+        //        }
+        //        return Collections.emptyMap();
```

Comment:
I think this change is not intentional, right?

Reply:
Oops!